VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx

```diff
@@ -0,0 +1,143 @@
+            {t('militaryInformation.history', { begin: service.formattedBeginDate, end: service.formattedEndDate })}
+          </TextView>
+        </Box>
+      </Box>
+    )
+  })
+
+  const branch = mostRecentBranch || ''
+
+  const boxProps: BoxProps = {
+    minHeight: 81,
+    borderRadius: 6,
+    pt: theme.dimensions.cardPadding,
+    pb: theme.dimensions.cardPadding,
+    backgroundColor: theme.colors.background.veteranStatus as BackgroundVariant,
+    borderTopWidth: theme.dimensions.borderWidth,
+    borderColor: theme.colors.border.veteranStatus as BorderColorVariant,
+    borderStyle: 'solid',
+  }
+
+  const getBranchSeal = (): React.ReactNode => {
+    const dimensions = {
+      width: 34,
+      height: 34,
+    }
+
+    switch (branch) {
+      case BranchesOfServiceConstants.AirForce:
+        return <VAIcon testID="United States Air Force" name="AirForce" {...dimensions} />
+      case BranchesOfServiceConstants.Army:
+        return <VAIcon testID="United States Army" name="Army" {...dimensions} />
+      case BranchesOfServiceConstants.CoastGuard:
+        return <VAIcon testID="United States Coast Guard" name="CoastGuard" {...dimensions} />
+      case BranchesOfServiceConstants.MarineCorps:
+        return <VAIcon testID="United States Marine Corps" name="MarineCorps" {...dimensions} />
+      case BranchesOfServiceConstants.Navy:
+        return <VAIcon testID="United States Navy" name="Navy" {...dimensions} />
+    }
+  }
+
+  return (
+    <LargePanel title={t('veteranStatus.title')} rightButtonText={t('close')}>
+      <Box backgroundColor={theme.colors.background.veteranStatus as BackgroundVariant} flex={1}>
+        <Box alignItems="center">
+          <Box flex={1} mt={theme.dimensions.standardMarginBetween}>
+            <VAIcon name={'Logo'} />
+          </Box>
+          {/* <Box my={theme.dimensions.standardMarginBetween}>
+          //TODO: Put back PhotoUpload later after concerns have been met
+            <PhotoUpload width={100} height={100} />
+          </Box> */}
+          <Box my={theme.dimensions.formMarginBetween}>
+            <TextView textTransform="capitalize" mb={theme.dimensions.textIconMargin} variant="BitterBoldHeading" color="primaryContrast">
+              {name()}
+            </TextView>
+            {accessToMilitaryInfo && (
+              <Box display="flex" flexDirection="row">
+                {getBranchSeal()}
+                <TextView ml={10} variant="MobileBody" color="primaryContrast">
+                  {branch}
+                </TextView>
+              </Box>
+            )}
+          </Box>
+        </Box>
+        <Box mx={theme.dimensions.gutter}>
+          {ratingIsDefined && (
+            <Box {...boxProps}>
+              <TextView variant="MobileBodyBold" color="primaryContrast">
+                {t('disabilityRating.title')}
+              </TextView>
+              <TextView variant="MobileBody" color="primaryContrast">
+                {combinedPercentText}
```

Comment:
This all may be fine due to the boxes, but questions about WCAG spacing and making sure these are properly applied, should probably confirm with Brea.

Reply:
Confirmed is fine with Brea.